pkg/volume/util/operationexecutor/operation_generator_test.go

```diff
@@ -89,12 +88,12 @@ func TestOperationGenerator_GenerateUnmapVolumeFunc_PluginName(t *testing.T) {
 			"volume_plugin":  expectedPluginName,
 		}
 		// compare the relative change of the metric because of the global state of the prometheus.DefaultGatherer.Gather()
```

Comment:
/cc @RainbowMango  
hello , i have some trouble by ussing  *AndCompare etc here.
here , we need to use them ? could you give me some help.Thank you very much.